wasm/Cargo.toml

```diff
@@ -77,9 +102,11 @@ version = "0.3.37"
 
 [features]
 default = [ "full" ]
-full = [ "console", "curves", "fields", "synthesizer", "utilities" ]
-console = [ "snarkvm-console" ]
+full = [ "console", "curves", "fields", "ledger", "synthesizer", "utilities" ]
+console = [ "snarkvm-console/default" ]
 curves = [ "snarkvm-curves" ]
 fields = [ "snarkvm-fields" ]
+ledger = [ "snarkvm-ledger-block", "snarkvm-ledger-query", "snarkvm-ledger-store" ]
+network = [ "snarkvm-circuit-network", "snarkvm-console/network" ]
```

Comment:
nit: move this into `circuit` and `console`, and update `full` to include `circuit`.